Program.cs

```diff
@@ -1,10 +1,15 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// Write a program to print a diamond using an asterisk
+// The input can be the number of rows (half the height).
+
+Console.Write ("Enter the number of rows: ");

```

Comment:
Fix this error.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/b413c40f-7b91-42d7-bde6-250d2b58203b)